marray/__init__.py

```diff
@@ -577,3 +575,14 @@ def var(x, axis=None, correction=0, keepdims=False):
                 pass
 
     return mod
+
+
+def _xinfo(x):
+    xp = x._xp
+    if xp.isdtype(x.dtype, 'integral'):
+        return xp.iinfo(x.dtype)
+    elif xp.isdtype(x.dtype, 'bool'):
+        binfo = dataclasses.make_dataclass("binfo", ['min', 'max'])
+        return binfo(min=False, max=True)
+    else:
+        return xp.finfo(x.dtype)
```

Comment:
if this ever became more complicated than just `min` and `max`, it could probably make its way into array-api-extra

Reply:
Sure. IIRC, I've wanted a dtype-independent `info` in SciPy before, too.